xmm-pysas-intro-short.ipynb

```diff
@@ -0,0 +1,212 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "e10d847e-8f3d-4ad1-b0d4-c2427a7a7002",
+   "metadata": {},
+   "source": [
+    "# (Short) XMM-Newton pySAS Introduction\n",
+    "---\n",
+    "This tutorial provides a short, basic introduction to using pySAS on SciServer. A much more comprehensive introduction can be found in the <a href=\"./xmm-pysas-intro-long.ipynb\">Long pySAS Introduction</a> In this tutorial we will demonstrate,\n",
```

Comment:
It is probably a good idea to put a link to some general documentation (.e.g. ABC guide) for users who are just starting an may not have enough idea what ODF, SAS etc are.